### PR TITLE
Fixes or Complicates Bug 1219538 - untested proposal

### PR DESCRIPTION
**Pull request by twobraids**
I saw this bug go by and thought it could be resolved fairly easily.   It's hard to test without actually putting it in place and try it...

### CODE REVIEW

File: scripts/crons/cron_libraries.sh [hunk 1]
```diff
@@ -47,53 +46,29 @@ techo "Processing for DATE: $DATE and WEEK: $WEEK"
...
-    techo "per-crash-interesting-modules.py > $TMPDIR/${DATE}_${I}_${J}-interesting-addons.txt"
-    $PYTHON /data/crash-data-tools/per-crash-interesting-modules.py -a -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-interesting-addons.txt
-    techo "per-crash-interesting-modules.py > $TMPDIR/${DATE}_${I}_${J}-interesting-addons-with-versions.txt"
-    $PYTHON /data/crash-data-tools/per-crash-interesting-modules.py -v -a -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-interesting-addons-with-versions.txt
-    techo "Phase 1: Version: $J end"
-  done
-  techo "Phase 1: end"
+    techo "Phase 1: Product: $I"
+    techo "Running psql query for version list."
+  VERSIONS=`psql -t -U $SECRETS_POSTGRESQL_DATABASE_USERNAME -h $RESOURCE_POSTGRESQL_DATABASE_HOSTNAME breakpad -c "select version_string from product_versions where product_name = '${I}' and '${SQL_DATE}' between build_date and sunset_date" | awk '{print $1}'`
```
Comment (lonnen) [inline scripts/crons/cron_libraries.sh]: I asked KaiRo in IRC to confirm how versions are chosen. I can't remember if its all active versions or featured versions. If you've already done this bit, please ignore this comment.

File: scripts/crons/cron_libraries.sh [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
 
-PGPASSWORD=$databasePassword
+PGPASSWORD=$SECRETS_POSTGRESQL_DATABASE_PASSWORD
 export PGPASSWORD
 
 DATE=`date '+%Y%m%d'`
 WEEK=`date -d 'last monday' '+%Y%m%d'`
 
-TMPDIR=`mktemp -d`
+TMPDIR=`mktemp -d -p /mnt/crashanalysis/crash_analysis/tmp`
```
Comment (lonnen) [inline scripts/crons/cron_libraries.sh]: Why provide the directory here?

**Comment by peterbe** [comment]
Can this be closed?

**Comment by peterbe** [comment]
This got wrapped up in https://github.com/mozilla/socorro/pull/3163 and merged.

**Pull request closed without merging**